app/assets/javascripts/test/backend-snapshot-tests/annotations.e2e.js

```diff
@@ -150,7 +150,7 @@ async function sendUpdateActions(explorational, queue) {
   const skeletonTracingId = explorational.tracing.skeleton;
   if (skeletonTracingId == null) throw new Error("No skeleton tracing present.");
   return sendRequestWithToken(
-    `${explorational.dataStore.url}/data/tracings/skeleton/${skeletonTracingId}/update?token=`,
+    `${explorational.dataStore.url}/tracings/skeleton/${skeletonTracingId}/update?token=`,
```

Comment:
Shouldn't this be the tracingstore url? But the e2e tests go through successfully - I'm confused ^^

Reply:
It's probably because both are the same, right? But the tracingstore url would be semantically correct.